hack/install-ckcp.sh

```diff
@@ -18,19 +18,27 @@ if [ "$1" == "--destroy" ]; then
   oc delete project spi-vault
 fi
 
+set -e
+
```

Comment:
we could move it to the very top of the script

Reply:
moved and disabled for destroy part